File: C/2540.c
#include <stdio.h>
int main() {
    int n, i, x;
    double voto=(100.0/3)*2, cont=0.0, porc;

    while(scanf("%d", &n)!=EOF) {
        for(i=0; i<n; i++) {
            scanf("%d", &x); 
            if (x==1) cont++;
        }
        porc=(100*cont)/n;
        if (porc>=voto) printf("impeachment\n");
        else printf("acusacao arquivada\n");
        cont=0;
    }
    return 0;
}

File: Python/2540.py
favor=0
contra=0
while True:
    try:
        n=int(input())
        linha=input().split(" ")
        for i in range(n):
            x=int(linha[i])
            if (x==1): favor+=1
            elif (x==0): contra+=1
        if (favor/(favor+contra)>=2/3): print("impeachment")
        else: print("acusacao arquivada")
        favor=0
        contra=0
    except (EOFError):
        break